dialogflow/analyze_content_stream.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Dialogflow API sample code using the StreamingAnalyzeContent
+API.
+
+Also please contact Google to get credentials of this project and set up the
+credential file json locations by running:
+export GOOGLE_APPLICATION_CREDENTIALS=<cred_json_file_location>
+
+Example usage:
+    export GOOGLE_CLOUD_PROJECT='cloud-contact-center-ext-demo'
+    export CONVERSATION_PROFILE='FnuBYO8eTBWM8ep1i-eOng'
+    export GOOGLE_APPLICATION_CREDENTIALS='/Users/ruogu/Desktop/keys/cloud-contact-center-ext-demo-78798f9f9254.json'
+    export AUDIO_FILE_PATH='/Users/ruogu/Desktop/book_a_room.wav'
+
+    python analyze_content_stream.py
```

Comment:
I understand this might have been copied from another sample, but this is not valid for any other user that isn't you/the original author.